shared/oae/api/oae.api.util.js

```diff
@@ -1011,7 +1017,7 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
                 // The `emptyText` is the text that will be shown when no suggested items could be found.
                 // If no `emptyText` has been provided, we fall back to a default string unless email
                 // addresses are allowed
-                if (!options.emptyText && !options.allowEmail) {
+                if (!options.emptyText && (!options.allowEmail || !allowInvitingGuests)) {
```

Comment:
Should this just check for `options.showResultListWhenNoMatch` instead of `(!options.allowEmail || !allowInvitingGuests)` ?